### PR TITLE
Add the `PubHeader` to the `SpubHeader`

### PR DESCRIPTION
**Pull request by idreyn**
We did it!

Test plan:
Verify that
- The `PubHeader` is visible when the "Preview" tab is visible
- The Collections bar is hidden
- The title and byline are read-only
- The "create Release" and "request review" buttons are not visible (the screenshot below has the Release button because I'm logged in as the superadmin)
- The header sticks to the top of the screen when scrolling 

image

### CODE REVIEW

File: client/containers/Pub/SpubHeader/spubHeader.scss [hunk 1]
```diff
@@ -1,72 +1,24 @@
...
-@import 'styles/variables.scss';
-$light-grey: #BDBDBD;
-$lighter-grey: #747474;
-// $spub-configuration-width: 1000px;
-// $title-elements-spacing: 15px;
+@import '../PubHeader/pubHeader.scss';
+@import './styles.scss';
 
 .spub-header-component {
-	&,.minimal-editor-component > .editor-wrapper {
```
Comment (idreyn) [inline client/containers/Pub/SpubHeader/spubHeader.scss:9]: I moved all of this stuff to the `SpubHeaderTab` component so it wouldn't affect the `PubHeader`.

File: client/containers/Pub/SpubHeader/SpubHeaderTab.tsx [hunk 2]
```diff
@@ -18,7 +18,7 @@ const SpubHeaderTab = (props: Props) => {
 	const [minHeight, setMinHeight] = useState(0);
 	const { viewportHeight } = useViewport();
 
-	useEffect(() => {
+	useLayoutEffect(() => {
```
Comment (howebenhowe) [inline client/containers/Pub/SpubHeader/SpubHeaderTab.tsx:21]: I deliberated much when choosing `useEffect` over `useLayoutEffect` and I either never understood or forgot my rationale. Would love to know what changed when you swapped for the latter.

File: client/containers/Pub/PubHeader/TitleGroup.tsx [hunk 3]
```diff
@@ -4,29 +4,29 @@ import { PubByline, DialogLauncher, PubAttributionDialog } from 'components';
...
 
 const TitleGroup = (props: Props) => {
 	const { pubData, updatePubData } = props;
 	const { title, description, isRelease } = pubData;
 	const { communityData, scopeData } = usePageContext();
+	const { submissionState } = usePubContext();
+	const isUnsubmitted = submissionState?.submission.status === 'incomplete';
 	const { canManage } = scopeData.activePermissions;
-	const canModify = canManage && !isRelease;
+	const canModify = canManage && !isRelease && !isUnsubmitted;
```
Comment (howebenhowe) [inline client/containers/Pub/PubHeader/TitleGroup.tsx:25]: This makes for the peculiar experience of the pub title + byline suddenly becoming editable from within the preview view. A UX question, I don't have strong sense of what to do about it.

**Review by howebenhowe — APPROVED** [review]
Looks right, smells right.
Have a couple questions for you entailed within, but LGTM!

**Review by qweliant — APPROVED** [review]

**Pull request merged**